### ISSUE TITLE
FP16 inference

### ISSUE THREAD

**Issue by Justin020718** [microsoft/DCVC#58]
Thank you for your great effort in building DCVC series models.

A few month earlier, I've achieved about 13 fps (DCVC-DC with fp16 precision, decoding only) in TensorRT with 2K videos, and for 720P videos, it can achieve 60fps (DCVC-DC with fp16 precision, decoding only). I've only checked quality of reconstructed I frame image, and it was just fine. When the precision goes down to 8 bit, the reconstructed image suffers from a huge quality loss. 

I'm trying to do the same thing based on DCVC-FM models. Now you've implemented new "block_mc_kernel", can it work on 8 bit precision? What should I do to use your implementation in ONNX or TensorRT? My conversion work flow is pytorch->ONNX->TensorRT.

Also, I'm interested in doing some fine tunings based on DCVC-FM, could you please send me the training code? I can send you a tiny but robust rate control algorithm (based on DCVC-FM) in return. My email is 2271516980@qq.com.
